app/src/main/java/com/glowstudio/android/blindsjn/feature/paymanagement/PayManagementScreen.kt

```
@@ -218,17 +403,38 @@ fun PayManagementScreen(
+                                    tint = TextSecondary
+                                )
+                            }
+                        }
                         Spacer(Modifier.height(8.dp))
                         Row(Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceBetween) {
                             Column(horizontalAlignment = Alignment.CenterHorizontally) {
                                 Text("고정비", fontSize = 14.sp, color = TextSecondary)
                                 Spacer(Modifier.height(4.dp))
-                                Text("₩ 1,200,000", fontWeight = FontWeight.Bold, fontSize = 16.sp, color = TextPrimary)
+                                Text(
+                                    "₩ ${String.format("%,d", viewModel.fixedCost.collectAsState().value.toInt())}",
```

detekt (1.23.8)

[warning] String.format("%,d", viewModel.fixedCost.collectAsState().value.toInt()) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)